libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -785,6 +793,7 @@ TYPED_TEST(BasicArithmeticTest, Division)
   TypeParam zero(0);
   TypeParam one(1);
   TypeParam two(2);
+  TypeParam three(3);
```

Comment:
"TypeParam const" would be better if you don't plan to change this variables

Reply:
yes, I plan to do a massive change for that and the constructors -to use the {} construtor- in all these tests, but not in this PR